test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -52,6 +52,9 @@ describe("Bitcoind", () => {
     const { chain_balance: initialBalance } = await getChainBalance({ lnd: lnd1 })
     const sats = initialBalance + btc2sat(amount)
 
+    // initiate the dealer wallet
+    await getUserWallet(6)
```

Comment:
nothing to be fixed now but maybe we need to add some helpers for funder and dealer wallets

Reply:
agree